examples/demo.ipynb

```diff
@@ -5,6 +5,15 @@
    "execution_count": 1,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Attempted file (/Users/danielt/Downloads/AICS-12_821.ome.tif) load with reader: aicsimageio.readers.bfio_reader.OmeTiledTiffReader failed with error: No module named 'bfio'\n",
```

Comment:
is this error a problem?

Reply:
No - this is more like a warning related to aicsimageio and really has no bearing on this PR.  I can probably reset the notebook to clear that out.  In the example, that message gets written out but the data still loads.